config/locales/application_form.yml

```
@@ -705,6 +705,7 @@ en:
             award_year:
               blank: Enter year the assessment was done
               invalid: Enter a real year
+              in_the_future: Enter a year before %{date}
```

I think the error message should reference what the year is ("assessment year"?), and perhaps doesn't need to include the next year?

```suggestion
              in_the_future: Assessment year must be this year or a previous year
```

or

```suggestion
              in_the_future: Assessment year must not be in the future
```